### PR TITLE
Add beta fence for issue #14692

### PR DESCRIPTION
**Pull request by teddyjtorres**
For issue #14692

### CODE REVIEW

**Comment by teddyjtorres** [comment]
#build

File: dev/com.ibm.ws.security.openidconnect.client/src/com/ibm/ws/security/openidconnect/client/web/OidcRedirectServlet.java [hunk 1]
```diff
@@ -144,7 +149,7 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
             return;
         }
 
-        if (OidcClientUtil.isReferrerHostValid(request, requestUrl) == false) {
+        if (isRedirectionUrlValid(request, requestUrl) == false) {
```
Comment (tevans78) [inline dev/com.ibm.ws.security.openidconnect.client/src/com/ibm/ws/security/openidconnect/client/web/OidcRedirectServlet.java:152]: Previously, this `if` block did not exist so there would never be any kind of error or exception. Now, if not beta (i.e. GA), you will get a UnsupportedOperationException. I think you need to change this beta guard to simply skip this block if not in beta.
```
if (ProductInfo.getBetaEdition()) {
   if (isRedirectionUrlValid(request, requestUrl) == false) {
   ...
   }
}
```

**Review by tevans78 — CHANGES_REQUESTED** [review]
I don't think the guard is quite right

File: dev/com.ibm.ws.security.openidconnect.client/src/com/ibm/ws/security/openidconnect/client/web/OidcRedirectServlet.java (hunk 1 above)
Comment (teddyjtorres) [reply dev/com.ibm.ws.security.openidconnect.client/src/com/ibm/ws/security/openidconnect/client/web/OidcRedirectServlet.java:152]: @tevans78  Pushed new commit to skip the check, but still issue the beta message on first call.

**Comment by teddyjtorres** [comment]
#build

**Review by tevans78 — APPROVED** [review]

**Review by ayoho — APPROVED** [review]

**Comment by teddyjtorres** [comment]
The build was good, but some of the checks were not and the code cannot be merged. I restarted the checks, but the failures do not seem related to the changes in this PR. I would need from someone who can assist in understanding the check failures.

**Comment by ayoho** [comment]
#libby

**Pull request merged**